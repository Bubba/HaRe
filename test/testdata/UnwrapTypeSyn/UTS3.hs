module UTS3 where

type Foo = String

type Bar = (Int, Int)

addSpaces :: Foo -> Bar -> Foo
addSpaces str (row, col) = str ++ (take row rows) ++ (take col cols)
  where rows = repeat '\n'
        cols = repeat ' '
