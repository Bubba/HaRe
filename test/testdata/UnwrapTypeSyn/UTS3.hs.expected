module UTS3 where

type Foo = String

addSpaces :: Foo -> (Int, Int) -> Foo
addSpaces str (row, col) = str ++ (take row rows) ++ (take col cols)
  where rows = repeat '\n'
        cols = repeat ' '